app/views/support/WeeklyPromotion.scala

```diff
@@ -137,13 +137,13 @@ object WeeklyPromotion {
       plan <- plans
       sixOrDiscount = isSixWeek(plan) || maybePromotion.exists(_.appliesTo.productRatePlanIds.contains(plan.id))
       pretty = maybePromotion.filter(_.appliesTo.productRatePlanIds.contains(plan.id)).flatMap(_.asDiscount)
-        .map { discountPromo => plan.charges.prettyPricingForDiscountedPeriod[scalaz.Id.Id, WeeklyLandingPage](discountPromo, currency) }
+        .map { discountPromo => plan.charges.prettyPricingForDiscountedPeriod[scalaz.Id.Identity, WeeklyLandingPage](discountPromo, currency) }
```

Comment:
I'm not sure we should be using "Identity" as it's not supposed to be a covariant alternative to Id, it seems to be something else that doesn't quite make sense, and wasn't really confirmed by the scalaz people as even a good idea (but has been sitting in the code a while).  We could alternaticely implement our own in the style of David's CovariantNonEmptyList (a CovariantId) with the same implementation as the invariant one.
See my comments on https://github.com/guardian/membership-common/pull/558/files#r166326856